storage/database/sharded_database.go

```diff
@@ -428,6 +428,15 @@ func (db *shardedDB) Meter(prefix string) {
 	}
 }
 
+func (db *shardedDB) GetProperty(name string) string {
+	var buf bytes.Buffer
+	for index, shard := range db.shards {
+		buf.WriteString(fmt.Sprintf("shard %d:\n", index))
+		buf.WriteString(shard.GetProperty(name))
```

Comment:
The Shard index and its property will be written in different lines. Is that intended? For me,  
`buf.WriteString(fmt.Sprintf("shard %d: %s\n", index, shard.GetProperty(name)))` looks more intuitive.